### PR TITLE
Backbone 282/implement get datasets

### PR DESCRIPTION
**Pull request by Mikdaam**

### CODE REVIEW

File: src/main/java/fr/uge/lp/databackbonepoc/DatabackbonePocApplication.java [hunk 1]
```diff
@@ -2,11 +2,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 
 /**
  * Entry point of the app.
  */
-@SpringBootApplication
+@SpringBootApplication(exclude={DataSourceAutoConfiguration.class})
```
Comment (tvdu94) [inline src/main/java/fr/uge/lp/databackbonepoc/DatabackbonePocApplication.java]: exclude part is useless I think

File: src/main/java/fr/uge/lp/databackbonepoc/models/DataModel.java [hunk 2]
```diff
@@ -1,47 +1,34 @@
-package fr.uge.lp.databackbonepoc.entities;
+package fr.uge.lp.databackbonepoc.models;
 
-import fr.uge.lp.databackbonepoc.relationship.Relationship;
+import lombok.Data;
```
Comment (tvdu94) [inline src/main/java/fr/uge/lp/databackbonepoc/models/DataModel.java:4]: remove useless line?

File: src/main/java/fr/uge/lp/databackbonepoc/services/DataModelService.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+		var entities = new ArrayList<Entity>();
+		try (var connection = dataSource.getConnection()) {
+			var metaData = connection.getMetaData();
+
+			var tables = metaData.getTables(null, "dbo", datasetName + "_%"+dataModelName+"_%", new String[]{"TABLE"});
+			while (tables.next()) {
+				var attributes = new ArrayList<Attribute>();
+				var entityName = tables.getString("TABLE_NAME");
+				var columnsResultSet = metaData.getColumns(null, "dbo", entityName, null);
```
Comment (tvdu94) [inline src/main/java/fr/uge/lp/databackbonepoc/services/DataModelService.java:74]: maybe add a variable "schemaPattern" to avoid repetitions

**Review by tvdu94 — CHANGES_REQUESTED** [review]

File: src/main/java/fr/uge/lp/databackbonepoc/DatabackbonePocApplication.java (hunk 1 above)
Comment (Mikdaam) [reply src/main/java/fr/uge/lp/databackbonepoc/DatabackbonePocApplication.java]: done

File: src/main/java/fr/uge/lp/databackbonepoc/models/DataModel.java (hunk 2 above)
Comment (Mikdaam) [reply src/main/java/fr/uge/lp/databackbonepoc/models/DataModel.java:4]: done

File: src/main/java/fr/uge/lp/databackbonepoc/services/DataModelService.java (hunk 3 above)
Comment (Mikdaam) [reply src/main/java/fr/uge/lp/databackbonepoc/services/DataModelService.java]: done

**Review by tvdu94 — APPROVED** [review]

**Pull request merged**